src/components/Dropdowns/ColorRampDropdown.tsx

```diff
@@ -197,11 +199,12 @@ export default function ColorRampSelection(inputProps: ColorRampSelectionProps):
     return selectedRamp.createGradientCanvas(DROPDOWN_WIDTH_PX, theme.controls.height).toDataURL();
   }, [props.selectedRamp, props.reversed]);
 
+  const showAsReversed = selectedRampData.reverseByDefault === true ? !props.reversed : props.reversed;
```

Comment:
Don't need `=== true`:
```suggestion
  const showAsReversed = selectedRampData.reverseByDefault ? !props.reversed : props.reversed;
```
...in fact, if you wanted to get extra cute with it, this is equivalent to:
```suggestion
  const showAsReversed = props.reversed !== selectedRampData.reverseByDefault;
```